### ISSUE TITLE
Feature Request: Indicate when inactive

### ISSUE THREAD

**Issue by brumac** [WarmUpTill/SceneSwitcher#25]
For most purposes, writing 
Inactive
to the same file that holds the current scene would work.

Perhaps tag the Current scene in that file with an leading underscore or whatever

If able to write to OBS' log, that would work, but that would mean finding the last entry in the log.

Fantastic addition to OBS -- thanks

**Comment by WarmUpTill** [comment]
I added writing start/stop info to the OBS log with 5ab8199e0338ae79620f91c09fe2db0b51c62529

**Comment by macgiver510** [comment]
Is it possible to add an indicator entry to the status bar where the LIVE / REC and CPU indicators are displayed?

Like 
ASS: <active>/<inactive>

By hovering over a short text that ASS means Advanced scene switcher

**Comment by WarmUpTill** [comment]
>Is it possible to add an indicator entry to the status bar where the LIVE / REC and CPU indicators are displayed?

Unfortunately I do not think this is possible with the OBS API at them moment

**Comment by brumac** [comment]
@WarmUpTill  
I installed release 1.61 just now to give it a quick check.
The following appeared in the log:
`19:06:09.696: LoadLibrary failed for '../../obs-plugins/64bit/advanced-scene-switcher.dll': The specified module could not be found.
19:06:09.696:  (126)
19:06:09.696: Module '../../obs-plugins/64bit/advanced-scene-switcher.dll' not loaded
`
I reverted to release 1.6 which still works.
No indication of which module it is looking for since obs-studio clearly saw the dll
